prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -646,9 +646,13 @@ void rxr_pkt_handle_send_error(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entr
 	assert(pkt_entry->alloc_type == RXR_PKT_FROM_EFA_TX_POOL ||
 	       pkt_entry->alloc_type == RXR_PKT_FROM_SHM_TX_POOL);
 
-	FI_WARN(&rxr_prov, FI_LOG_CQ,
-			"Packet send error: %s (%d)\n",
-			efa_strerror(prov_errno), prov_errno);
+	/* Avoid excessive warn logging */
+	if (err != FI_EAGAIN || prov_errno != FI_EFA_REMOTE_ERROR_RNR) {
+		FI_WARN(&rxr_prov, FI_LOG_CQ,
+				"Packet send error: %s (%d)\n",
+				efa_strerror(prov_errno), prov_errno);
```

Comment:
I think we should still log RNR in the info/debug level.

Reply:
I am not sure.

I do not think we need to print anything here. For real error, this function will call `rxr_cq_write_rx_error` and `rxr_cq_write_tx_error`, which printed the same information. So I do not think it is useful to print here.

Reply:
So just remove altogether?

Reply:
Removal would be my preference, or change to `FI_DEBUG` like @shijin-aws suggested.

Reply:
Then let's just change `FI_WARN` to `FI_DEBUG` for these two prints.